x-pack/plugins/ml/public/application/components/ml_inference/add_inference_pipeline_flyout.tsx

```diff
@@ -184,6 +185,7 @@ export const AddInferencePipelineFlyout: FC<AddInferencePipelineFlyoutProps> = (
           isConfigureProcessorStepValid={hasUnsavedChanges === false}
           pipelineCreated={formState.pipelineCreated}
           creatingPipeline={formState.creatingPipeline}
+          hasProcessorStep
```

Comment:
Is it possible to force a refresh of the trained models page when the flyout is closed after creating a pipeline? Currently I have to manually hit the Refresh button to see the indicator update on the Pipelines tab.

<img width="115" alt="image" src="https://github.com/elastic/kibana/assets/7405507/d508c4bb-ebaa-4a9a-87cd-bee4e540ba6c">

Reply:
Pipelines tab looks to be updating correctly with the changes in https://github.com/elastic/kibana/pull/170902/commits/869c46cd1958becccee926172373b5e52d125d87